### ISSUE TITLE
Redundant "Attributes" page

### ISSUE THREAD

**Issue by devs4v** [jquery/learn.jquery.com#642]
There's a section on this page: http://learn.jquery.com/using-jquery-core/manipulating-elements/ which has the "manipulating attributes" section.

However, there's a standalone page earlier ( http://learn.jquery.com/using-jquery-core/attributes/ ) with the same content, minus the code where it gives you an example of how one can dynamically generate the values of the attributes.

Is that intentionally done so or just redundant? I guess latter page (attributes standalone page) can be merged into the manipulating elements page.

Moreover, the "Attributes" page does not define what a getter and setter is (though it gives a really good example for it), but the same is defined in a subsequent page ( http://learn.jquery.com/using-jquery-core/working-with-selections/ ) with examples. Hence, IMHO, reordering or merging the Attributes page seems like a good idea.

**Comment by arthurvr** [comment]
Yeah, these pages are in some ways redundant, I agree. Will work on it once I get a little of time. Thanks for letting us know!
